test/e2e_tests/specs/ClearConversationContent/clearConversationContent.spec.ts

```diff
@@ -0,0 +1,400 @@
+/*
+ * Wire
+ * Copyright (C) 2025 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {User} from 'test/e2e_tests/data/user';
+import {PageManager} from 'test/e2e_tests/pageManager';
+import {test, withLogin, withConnectedUser, expect} from 'test/e2e_tests/test.fixtures';
+
+import {getTextFilePath, shareAssetHelper} from '../../utils/asset.util';
+import {getImageFilePath} from '../../utils/sendImage.util';
+import {createGroup} from '../../utils/userActions';
+
+test.describe('Clear Conversation Content', () => {
+  let userA: User;
+  let userB: User;
+  let userC: User;
+
+  test.beforeEach(async ({createTeam}) => {
+    const team = await createTeam('Test Team', {withMembers: 2});
+    userA = team.owner;
+    userB = team.members[0];
+    userC = team.members[1];
+  });
+
+  test(
+    'I want to clear content of a group conversation via conversation list',
+    {tag: ['@TC-152', '@regression']},
+    async ({createPage}) => {
+      const [userAPageManager, userBPageManager, userCPageManager] = await Promise.all([
+        PageManager.from(createPage(withLogin(userA), withConnectedUser(userB), withConnectedUser(userC))),
```

Comment:
Why are you connecting the users? They don't communicate via 1on1 conversations in here, so there's no need to connect them. ;)

```suggestion
        PageManager.from(createPage(withLogin(userA))),
```

Reply:
(Same applies to following tests)